app/constants/userOnboarding.js

```diff
@@ -1,12 +1,14 @@
-export const FormTitlesAndSubtitles = [
-    {
-      title: 'Welcome to HapDay',
-      subtitle:
-        'Tell us a bit about yourself, we will need this to get your profile setup. You’ll be able to edit this later.',
-    },
-    {
-      title: 'Connect your calendar',
-      subtitle:
-        'Connect your calendar to automatically check for busy times and new events as they’re scheduled.',
-    },
-  ];
\ No newline at end of file
+const FormTitlesAndSubtitles = [
```

Comment:
Why converted to default export?

Reply:
+1

Reply:
It was throwing Lint error due to which I updated it to Default export, Am I wrong here? Please check

Reply:
Changed back to the previous version 
Cc: @DeRaowl @iRohitGaur